lib/rules/cli/4.1.yaml

```diff
@@ -1112,6 +1112,12 @@
     :master-config: --master-config=<value>
     :node-config: --node-config=<value>
     :prevent-modification: --prevent-modification=<value>
+:oadm_inspect:
```

Comment:
@wewang58 please put the rules in alphabetic order.

Reply:
@pruan-rht Updated it, Thanks